buildSrc/src/main/java/com/google/firebase/gradle/plugins/ReleaseGenerator.kt

```diff
@@ -84,7 +84,7 @@ abstract class ReleaseGenerator : DefaultTask() {
       ReleaseConfig(currentRelease.get(), libsToRelease.map { it.path }.toSet())
     )
     val releaseReport = generateReleaseReport(changes, changedLibsWithNoChangelog)
-    if (printReleaseConfig.get().toBoolean()) {
+    if (printReleaseConfig.orNull.toBoolean()) {
       project.logger.info(releaseReport)
     }
```

Comment:
Are you talking about the `orNull`? If so, then yeah; the value could not be present (since it's an optional input). `orNull` will get the value, or return null if it's not present. `toBoolean()` converts `null` to `false`.